css-three-column-layout/styles.css

```diff
@@ -12,15 +12,7 @@
   background-color: #1896d9;
 }
 
-.column-one-third{
-  flex-basis: calc(100%/3);
-}
-
-.column-two-thirds{
-  flex-basis: calc(100%/3);
-}
-
-.column-third{
+.column{
```

Comment:
No, sorry, this is not what I intended. The name `column` give me no indication of its purpose. You need a column class for every size of `column` you use in your layout and a name that specifies its size. You should review the `css-layout-classes` exercise in depth to get a refresher on the concept.